compiler-rt/www/index.html

```diff
@@ -123,7 +123,7 @@ <h2>Get it and get involved!</h2>
   <li>cd llvm-project</li>
   <li>mkdir build-compiler-rt</li>
   <li>cd build-compiler-rt</li>
-  <li>cmake ../compiler-rt -DLLVM_CONFIG_PATH=/path/to/llvm-config</li>
+  <li>cmake ../compiler-rt -DLLVM_CMAKE_DIR=/path/to/llvm-project/cmake</li>
```

Comment:
```suggestion
  <li>cmake ../compiler-rt -DLLVM_CMAKE_DIR=/path/to/llvm-project/cmake/modules</li>
```

Reply:
`llvm-project/cmake` worked for me when I built standalone but I see elsewhere modules is also specified